pkgs/development/compilers/llvm/common/clang-tools/default.nix

```diff
@@ -10,50 +13,90 @@
   enableLibcxx ? false,
 }:
 
-stdenv.mkDerivation {
-  unwrapped = clang-unwrapped;
+let
+  clang-tools = stdenv.mkDerivation {
+    unwrapped = clang-unwrapped;
 
-  pname = "clang-tools";
-  version = lib.getVersion clang-unwrapped;
-  dontUnpack = true;
-  clang = if enableLibcxx then libcxxClang else clang;
+    pname = "clang-tools";
+    version = lib.getVersion clang-unwrapped;
+    dontUnpack = true;
+    clang = if enableLibcxx then libcxxClang else clang;
 
-  installPhase = ''
-    runHook preInstall
+    installPhase = ''
+      runHook preInstall
 
-    mkdir -p $out/bin
+      function wrapTool {
+        local tool="$1"
+        local toolName="$2"
 
-    for tool in $unwrapped/bin/clang-*; do
-      tool=$(basename "$tool")
+        cp $tool $out/bin/$toolName.unwrapped
```

Comment:
Usually (with e.g. `makeWrappe`) we call this `.$toolName-wrapped`.  I think we should follow the same convention here.

Reply:
kk, done